src/profile/v100/address/address.rs

```diff
@@ -0,0 +1,26 @@
+use crate::{address_union, prelude::*};
+
+address_union!(
```

Comment:
@micbakos-rdx this is all it takes to declare a tagged union of addresses, with ALL methods :)